test/publish/test_publications_by_author.py

```diff
@@ -313,6 +321,57 @@ def test_sulpub_fields(test_session, sulpub_json):
         assert publication._citation_count(pub_row) is None
 
 
+def test_volume_issue():
+    """
+    Test both volume and issue distill rules.
+    """
+    pub = Publication(
+        doi="10.000/example",
+        title="I'm a Book",
+        openalex_json={"biblio": {"volume": "1", "issue": "2"}},
+        dim_json={"volume": "3", "issue": "4"},
+        pubmed_json={
+            "MedlineCitation": {
+                "Article": {"Journal": {"JournalIssue": {"Volume": "5", "Issue": "6"}}}
+            }
+        },
+        sulpub_json={"journal": {"volume": "7", "issue": "8"}},
+    )
+
+    # slowly peel away the platform metadata that's available to confirm that we are
+    # matching in the right order, and looking for values correctly
+
+    assert publication._volume(pub) == "1"
+    assert publication._issue(pub) == "2"
+
+    pub.openalex_json = {}
+    assert publication._volume(pub) == "3"
+    assert publication._issue(pub) == "4"
+
+    pub.dim_json = {}
+    assert publication._volume(pub) == "5"
+    assert publication._issue(pub) == "6"
+
+    pub.pubmed_json = {}
+    assert publication._volume(pub) == "7"
+    assert publication._issue(pub) == "8"
+
+    pub.sulpub_json = {}
+    assert publication._volume(pub) is None
+    assert publication._issue(pub) is None
+
+
```

Comment:
This really shouldn't happen (a list of volumes or issues). But technically it's possible, so test coverage complained until I added a test for it.